### PR TITLE
Added email icon for email field

### LINKED ISSUE #1768: Improve the UI of email field
**Actual Behaviour**
The email field looks good but can be made better by adding a icon there.
**Expected Behaviour**
The email field should have a email icon.
**Screenshots of the issue**
screen shot 2018-03-10 at 1 09 48 pm

**Would you like to work on the issue?**
YES

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Added email icon in the email field.
screen shot 2018-03-10 at 5 44 51 pm

Fixes #1768

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@ParthS007 I'm unable to deploy a link using heroku.

**Comment by harsh-jindal** [comment]
@agbilotia1998 Okay

File: src/www/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta content="width=device-width, initial-scale=1" name="viewport">
   <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
   <link href='https://fonts.googleapis.com/css?family=Roboto:400,100' rel='stylesheet' type='text/css'>
+  <link href='https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css' rel='stylesheet' type='text/css'>
```
Comment (geekyd) [inline src/www/index.html]: We should avoid using external css over network.

**Comment by harsh-jindal** [comment]
@geekyd Shall i download the font awesome file and then add?

**Comment by mariobehling** [comment]
@harsh-jindal I am seeing PR is inactive. Please open a new PR if you decide to come back to this. Please avoid external CSS and add anything you would need into the repo/project itself. Cheers.

**Pull request closed without merging**